MQTicketService/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN cd /data/TicketGenerator && \
 # Tested with
 #     9.2.4.0-r1
```

Comment:
This tested statement is incorrect as 9.2.4.0-r1 is not available from icr.io/ibm-messaging/mq:latest